app/assets/stylesheets/active_admin/structure/_footer.scss

```diff
@@ -1,14 +1,9 @@
 #footer {
-  padding: 30px 30px;
-  font-size: 0.8em;
-  clear: both;
-
-  p {
-    padding-top: 10px
-  }
+  font-size: 0.9em;
+  padding: $horizontal-page-margin;
 }
 
 // -------------------------------------- Index Footer (Under Table)
-#index_footer { padding-top: 5px; text-align: right; font-size: 0.85em; }
-
-.index_content { clear: both; }
+#index_footer {
```

Comment:
Avoid using id selectors<br>Selector `index_footer` should be written in lowercase with hyphens